src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppAppRunner.recipe

```diff
@@ -310,16 +346,16 @@
         {
             "Id": "HealthCheckTimeout",
             "Name": "Health Check Timeout",
+            "Category": "Health",
             "Description": "The time, in seconds, to wait for a health check response before deciding it failed.",
             "Type": "Int",
             "DefaultValue": 2,
             "AdvancedSetting": true,
             "Updatable": true,
             "Validators": [
                 {
-                    "ValidatorType":"Range",
-                    "Configuration": 
-                    {
+                    "ValidatorType": "Range",
+                    "Configuration": {
```

Comment:
nit: can you remove whitespace changes?

Reply:
This is just how VS auto formats when you make edits. I don't really want to fight with the format back and forth.